src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -238,6 +238,27 @@ void TilePyramid::update(const std::vector<Immutable<style::LayerPropertiesWrapp
     for (auto& pair : tiles) {
         pair.second->setShowCollisionBoxes(parameters.debugOptions & MapDebugOptions::Collision);
     }
+
+    // Initialize render tiles fields and update the tile contained layer render data.
+    for (RenderTile& renderTile : renderTiles) {
+        Tile& tile = renderTile.tile;
+        if (!tile.isRenderable()) continue;
+
+        const bool holdForFade = tile.holdForFade();
+        for (const auto& layerProperties : layers) {
+            const auto* typeInfo = layerProperties->baseImpl->getTypeInfo();
+            if (holdForFade && typeInfo->fadingTiles == LayerTypeInfo::FadingTiles::NotRequired) {
+                continue;
+            }
+            // Update layer properties for complete tiles; for incomplete just check the presence.
+            bool layerIsInTile = tile.isComplete() ? tile.updateLayerProperties(layerProperties)
```

Comment:
Maybe rename `layerIsInTile` to `layerIsRenderable`?

Reply:
`renderableByTile`